code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -1,3 +1,5 @@
+var/global/list/world_mouses = list()

```

Comment:
...Jesus christ, is this really necessary?

Also, there's a file with all of the global lists. It should be there.